src/epub/text/scanners-live-in-vain.xhtml

```diff
@@ -33,7 +33,7 @@
 				</blockquote>
 				<p>He heard Luci chuckle in the next room. He gloated over the sounds of her dress as she swished to the doorway.</p>
 				<p>She gave him her crooked little smile. “You sound all right. Are you all right, really?”</p>
-				<p>Even with this luxury of senses, he scanned. He took the flash-quick inventory which constituted his professional skill. His eyes swept in the news of the instruments. Nothing showed off scale, beyond the Nerve Compression hanging in the edge of “Danger.” But he could not worry about the Nerve box. That always came through Cranching. You couldn’t get under the wire without having it show on the Nerve box. Some day the box would go to <em>Overload</em> and drop back down to <em>Dead</em>. That was the way a haberman ended. But you couldn’t have everything. People who went to the Up-and-Out had to pay the price for Space.</p>
+				<p>Even with this luxury of senses, he scanned. He took the flash-quick inventory which constituted his professional skill. His eyes swept in the news of the instruments. Nothing showed off scale, beyond the Nerve Compression hanging in the edge of <em>Danger</em>. But he could not worry about the Nerve box. That always came through Cranching. You couldn’t get under the wire without having it show on the Nerve box. Some day the box would go to <em>Overload</em> and drop back down to <em>Dead</em>. That was the way a haberman ended. But you couldn’t have everything. People who went to the Up-and-Out had to pay the price for Space.</p>
```

Comment:
I think `<q>`, styled with italics, is more appropriate for all of the italics in this PR, as this is not emphasized text

Reply:
The same concept (the status of a reading) which is indicated as "Danger" here is indicated as _Danger_ elsewhere, though not absolutely consistently; see line 60. See _Overload_ indicated on lines 36, 60, 77, 190, for example. Should I change those to <q> instead of <em> as well?

Reply:
You have to decide if the word is being emphasized in the context in which it's being used. IMO none of the italics in this particular PR are indicating emphasized speech, they're merely indicating a special word to the reader and actually quotes could be just as valid a choice as italics. Therefore `<q>` is more appropriate. 

For other cases, look at the context and make that analysis.

Reply:
I've considered all instances, and none of the box readings seem like emphasis to me, just, as you say, a special word, so I'm using `<q>` rather than `<em>` there.